MMM-voice.js

```diff
@@ -220,10 +222,51 @@ Module.register('MMM-voice', {
             MM.getModules().enumerate((module) => {
                 module.hide(1000);
             });
+            // tell other modules all hidden
+            this.sendNotification('NOW_ASLEEP','[]')
         } else if (notification === 'SHOW') {
             MM.getModules().enumerate((module) => {
                 module.show(1000);
             });
+            // tell other modules all shown
+            this.sendNotification('NOW_AWAKE')
+				} else if (notification === 'SLEEP_HIDE') {
+						// sleep by hiding (energyStar monitors)
+            let self=this;
+            let list=[];
+            MM.getModules().enumerate((module) => {
+               // if the module is already hidden
+               if(module.hidden==true){
+                  // save it for wake up
+                  self.previously_hidden.push(module)
```

Comment:
please push identifier only instead of the whole module

Reply:
changed

Reply:
this didn't push the module, only the address of the module.. there was no clone..

Reply:
exactly and when you're not cloning bad things can happen between different modules. Also it's safer to check for the identifier as some users are running multiple instances of modules